packages/Webkul/Admin/src/DataGrids/CustomerDataGrid.php

```diff
@@ -158,6 +159,17 @@ public function addColumns()
                 return $html;
             },
         ]);
+
+        $this->addColumn([
+            'index'       => 'is_suspended',
+            'label'       => trans('admin::app.customers.customers.suspended'),
+            'type'        => 'boolean',
+            'filter-type' => 'suspended',
```

Comment:
Any use of this key?

Reply:
filter-type

Reply:
We need this suspended key for **filters** only not for data grid so using this key we have removed this from data grid and only added into filter.